Sources/Public/CalendarView.swift

```diff
@@ -972,6 +980,27 @@ extension CalendarView: UIScrollViewDelegate {
 
 }
 
+// MARK: WidthDependentIntrinsicContentHeightProviding
+
+extension CalendarView: WidthDependentIntrinsicContentHeightProviding {
+
+  // This is where we perform our width-dependent height calculation. See `DoubleLayoutPassHelpers`
+  // for more details about why this is needed and how it works.
+  func intrinsicContentSize(forHorizontallyInsetWidth width: CGFloat) -> CGSize {
+    // Force layout so that `visibleItemsDetails.maxMonthHeight` is calculated.
+    setNeedsLayout()
+    layoutIfNeeded()
+
+    guard let visibleItemsDetails = visibleItemsDetails else {
+      return CGSize(width: UIView.noIntrinsicMetric, height: UIView.noIntrinsicMetric)
+    }
+
+    let height = visibleItemsDetails.maxMonthHeight + visibleItemsDetails.heightOfPinnedContent
+    return CGSize(width: UIView.noIntrinsicMetric, height: height)
```

Comment:
note: no intrinsic width. `CalendarView` can resize freely in the horizontal direction, and the only thing that will change is the max month height (and therefore, the intrinsic content height) in response.

Resizing the calendar horizontally does not impact the width - for horizontal calendars, you'll either have extra space at the bottom or you'll clip some content, and for vertical calendars you'll just create a larger or smaller scrollable region by changing the height.